app/ide-desktop/lib/client/src/index.ts

```diff
@@ -223,8 +223,8 @@ class App {
             await logger.asyncGroupMeasured('Starting the application', async () => {
                 // Note that we want to do all the actions synchronously, so when the window
                 // appears, it serves the website immediately.
-                await this.startBackendIfEnabled()
                 await this.startContentServerIfEnabled()
+                await this.startBackendIfEnabled()
```

Comment:
Would it be possible to run those in parallel?
```suggestion
                await Promise.all([
                    this.startContentServerIfEnabled(),
                    this.startBackendIfEnabled()
                ])
```

Reply:
We can, but I think if the content server fails to start, we don't need the backend. That was the logic. Also, the logs look prettier (the error comes last with this approach, which is minor, though)

Reply:
Sounds fine to keep it ordered if it causes less frustration 👍